src/plugins/gr-he-common/core/shell.py

```diff
@@ -103,5 +103,35 @@ def _setup(self):
                 if not self.environment[ohostedcons.CoreEnv.TMUX_PROCEED]:
                     raise context.Abort('Aborted by user')
 
+        if (
+            not self.environment[ohostedcons.NetworkEnv.FORCE_IPV4]
+            and not self.environment[ohostedcons.NetworkEnv.FORCE_IPV6]
+        ):
+            self.logger.warning(
+                _('It has been detected that either --4 or --6 '
+                  'options were not used.\n '
+                  'On a dual stack environment, IPv6 will be selected '
+                  'and due to a limitation, hosted-engine will be deployed '
+                  'as IPv6 only.\n '
+                  'Make sure that DNS returns only IPv6 '
+                  'or change Java and imageio address resolution.')
+                # TODO: add a page link for changing Java and imageio config
```

Comment:
Can you please do this now, instead of a TODO comment? Just create a page with very little content, we can update it later.

Reply:
In my comment above, I didn't refer to "change Java and imageio address resolution" (about which I do not know enough), but to the TODO comment. If we have any doubts about the current text (with or without "change Java and imageio address resolution"), I'd rather "prepare beforehand" and add a "See also: ovirt.org/something". There, we can just copy current text perhaps giving slightly more details/context. In the future, if we want to add something there, this will be much easier than changing the text in the code.

Reply:
Right, but if we will not mention the Java/imageio DNS resolution, we don't need the TODO.